core/trino-main/src/main/java/io/trino/cost/StatisticRange.java

```diff
@@ -173,28 +175,6 @@ public StatisticRange addAndCollapseDistinctValues(StatisticRange other)
         return new StatisticRange(minExcludeNaN(low, other.low), maxExcludeNaN(high, other.high), newDistinctValues);
     }
 
-    private static double minExcludeNaN(double v1, double v2)
```

Comment:
Squash this with other commit that moves methods to `MoreMath`

Reply:
I prefer to keep it as separate commit